release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2775,6 +2775,53 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.15.42
+[id="ocp-4-15-42_{context}"]
+=== RHSA-2024:11562 - {product-title} 4.15.42 bug fix and security update
+
+Issued: 02 January 2025
+
+{product-title} release 4.15.42, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:11562[RHSA-2024:11562] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHBA-2024:11565[RHBA-2024:11565] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.42 --pullspecs
+----
+
+[id="ocp-4-15-42-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, when the webhook token authenticator was enabled and had the authorization type set to `None`, the {product-title} web console would consistently crash. With this release, a bug fix ensures that this configuration does not cause the {product-title} web console to crash. (link:https://issues.redhat.com/browse/OCPBUGS-46482[*OCPBUGS-46482*])
```

Comment:
For bugs that don't have any complexity to the fix (where you end up just restating the issue), you an always lean on "The issue is resolved with this release" or similar :)